codegen/templates/vec.rs.tera

```diff
@@ -1862,14 +1862,15 @@ impl {{ self_t }} {
         self.sub(rhs).abs().cmple(Self::splat(max_abs_diff)).all()
     }
 
-    /// Returns a vector with a length no less than `min` and no more than `max`
+    /// Returns a vector with a length no less than `min` and no more than `max`.
     ///
     /// # Panics
     ///
-    /// Will panic if `min` is greater than `max` when `glam_assert` is enabled.
+    /// Will panic if `min` is greater than `max`, or if either `min` or `max` is negative, when `glam_assert` is enabled.
     #[inline]
     #[must_use]
     pub fn clamp_length(self, min: {{ scalar_t }}, max: {{ scalar_t }}) -> Self {
+        glam_assert!(0 as {{ scalar_t }} <= min);
```

Comment:
This method is only used in float vectors so it can be 0.0 without the need for a cast

Reply:
ohh, sure 👍